system/Images/Handlers/BaseHandler.php

```diff
@@ -769,4 +769,17 @@ public function getHeight()
     {
         return ($this->resource !== null) ? $this->_getHeight() : $this->height;
     }
+
+    /**
+     * Clears image metadata.
+     *
+     * This method has no use in the GDHandler,
+     * since all the data are cleared automatically.
+     *
+     * GDHandler can't preserve the image metadata.
```

Comment:
I think comment can be more generic like: "Placeholder for clearing metadata implementation"

Reply:
Changed - thank you.